en/core-libraries/internationalization-and-localization.rst

```diff
@@ -60,6 +60,11 @@ is selected.
 
 The core strings messages extracted from the cakephp library can be stored
 separately in a file named **cake.po** in **src/Locale/**.
+The `CakePHP localized library <https://github.com/cakephp/localized>`_ houses
+translations for the client-facing translated strings in the core (the cake
+domain). To use these files, link or copy them into their expected location:
+**src/Locale/<locale>/cake.po**. If your locale is incomplete or incorrect,
```

Comment:
In hindsight I wonder if this would have needed ` instead of being bold as its a string/codepiece.